install/kubernetes/namespace.yaml

```diff
@@ -5,3 +5,9 @@ metadata:
   labels:
     istio-injection: disabled
 ---
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: istio-config
```

Comment:
this should work well with `helm template`, but not work at all with `helm install`.  Perhaps an upgrade in helm's `tiller` component permits it to manage more than one namespace now.  A version of helm would help here.

Have you tried this manually with the tiller instructions?